content/en/dashboards/widgets/check_status.md

```diff
@@ -28,12 +28,13 @@ Check status shows the current status or number of results for any check perform
 
 1. Select a previously created service check.
 2. Choose a reporting timeframe:
-  * Global Time
-  * The past 10 minutes
-  * The past 30 minutes
-  * The past hour
-  * The past 4 hours
-  * The past day
+    * Global Time
+    * The past 10 minutes
+    * The past 30 minutes
+    * The past hour
+    * The past 4 hours
+    * The past day
+    **Note:**  The check status widget is for displaying the latest status. It does not display historical data. Use the Global Time option for toggling between the past 10 minutes, the past 30 minutes, etc.
```

Comment:
```suggestion
    **Note:**  The check status widget displays the latest status. It does not display historical data. Use the Global Time option for toggling between different time frames.
```

Reply:
Thank you for looking into this @kayayarai ! "different time frames" could include historical data (although we did say it does not historical data so it should be clear already.) The only supported options there are "The past X" options like:

  * The past 10 minutes
  * The past 30 minutes
  * The past hour
  * The past 4 hours
  * The past day
  * The past 2 days (Uses data same as the past day)
  * The past week (Uses data same as the past day)
  * The past month (Uses data same as the past day)
  
  Is there a better way for us to refer to the set above?